### ISSUE TITLE
Does this require touchpad capable of absolute positioning?

### ISSUE THREAD

**Issue by Zireael07** [BackMountainDevil/touchpadAsNumpad#1]
Title

**Comment by BackMountainDevil** [comment]
Yes if you mean in the code. on my device, it just got absolute position. I do not test other device cz I do not have.
Which touchpad does not support  absolute position?

**Comment by Zireael07** [comment]
From a quick google, some Cirque and Alps touchpads.

Even not all Synaptics do (which is a bummer as it's an extremely popular maker)

**Comment by BackMountainDevil** [comment]
I am not fimiliar with those brand. I tried to figure out which brand is my touchpad and got wired microsoft(I think it is not true, cz microsoft device is not cheap).


That is wired if we can not get the abs pos, how can we know where user touch and which direction user going?&nbsp; and not even multifinger? A wired touchpoint is little red point on IBM thinkpad, that just got direction without abs pos.


The way to know the answer is borrow a devide and test
